src/routes/notifications/notifications.controller.spec.ts

```diff
@@ -237,4 +241,78 @@ describe('Notifications Controller (Unit)', () => {
         });
     });
   });
+
+  describe('DELETE /chains/:chainId/notifications/devices/:uuid/safes/:safeAddress', () => {
+    it('Success', async () => {
+      const uuid = faker.datatype.uuid();
+      const safeAddress = faker.finance.ethereumAddress();
+      mockNetworkService.get.mockImplementation((url) =>
+        url.includes(`${safeConfigUrl}/api/v1/chains/`)
+          ? Promise.resolve({ data: chainBuilder().build() })
+          : rejectForUrl(url),
+      );
+      mockNetworkService.delete.mockImplementation((url) =>
+        url.includes(
+          `/api/v1/notifications/devices/${uuid}/safes/${safeAddress}`,
+        )
+          ? Promise.resolve()
+          : rejectForUrl(url),
+      );
+
+      await request(app.getHttpServer())
+        .delete(
+          `/chains/${faker.random.numeric()}/notifications/devices/${uuid}/safes/${safeAddress}`,
+        )
+        .expect(200);
+      expect(mockNetworkService.delete).toBeCalledTimes(1);
```

Comment:
I'd add the payload expected to be used for this call.

Reply:
Done [d880fcb](https://github.com/5afe/safe-client-gateway-nest/pull/268/commits/d880fcb3732f5bd60894507bcb3104e4a46b7714)